I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/options.rpy", line 237, in script
    init python:
  File "game/options.rpy", line 237, in script
    init python:
  File "game/options.rpy", line 262, in <module>
    build.classify('**.rpyc')
TypeError: classify() missing 1 required positional argument: 'file_list'

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "C:\renpy\renpy\bootstrap.py", line 359, in bootstrap
    renpy.main.main()
  File "C:\renpy\renpy\main.py", line 541, in main
    renpy.game.context().run(node)
  File "game/options.rpy", line 237, in script
    init python:
  File "lib/python3.9/future/utils/__init__.py", line 444, in raise_
  File "game/options.rpy", line 237, in script
    init python:
  File "C:\renpy\renpy\ast.py", line 834, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "C:\renpy\renpy\python.py", line 1187, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "game/options.rpy", line 262, in <module>
    build.classify('**.rpyc')
TypeError: classify() missing 1 required positional argument: 'file_list'

Windows-10-10.0.19045 AMD64
Ren'Py 8.3.8.25031801+nightly
Patchmonk's Advanced Inventory System 1.2.7 1.2.7
Sun Apr 13 05:44:06 2025
